### PR TITLE
Fix type error when opening statistics page

### LINKED ISSUE #566: Error when opening Statistics Page
When navigating to Statistics the type error pops up, which interrupts the rendering of all pages and shows a blank white page! (it is no-go)

Screenshot 2024-06-04 at 13 49 19

### PR DESCRIPTION
**Pull request by ferishili**
This PR fixes #566
by properly async call (using dispatch!) to fetch the user info before fetching the statistics info!
This PR needs actual statistic data to make sure it works properly, but for the start the error is now gone, and the page should not be interrupted!

Screenshot 2024-06-04 at 14 08 26

### CODE REVIEW

**Comment by ziegenberg** [comment]
This also fixe #282

**Comment by ziegenberg** [comment]
_Note: ICLA has been filed (see https://github.com/opencast/opencast-admin-interface/pull/546#issuecomment-2147116625)_

**Review by ziegenberg — CHANGES_REQUESTED** [review]

File: src/components/statistics/Statistics.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Header from "../Header";
 import Footer from "../Footer";
 import MainNav from "../shared/MainNav";
 import TimeSeriesStatistics from "../shared/TimeSeriesStatistics";
+import { addNotification } from "../../slices/notificationSlice";
```
[the remark this answers is not in the document]
Comment (ferishili) [reply src/components/statistics/Statistics.tsx:10]: You are right, I meant to provide error notification but it was already taken care of!
Removing it.

**Comment by ferishili** [comment]
Keep in mind that I haven't test this for #282 (which happens it events details level)
I found this bug simply by navigating to the statistics menu in left pane action bar!

**Review by ziegenberg — APPROVED** [review]
Does what it says on the tin

**Pull request merged**